docs/test-pr-guide.md

```diff
@@ -23,6 +23,40 @@ git pull https://github.com/<contributor>/mentorship-backend/ <branch-name>
 
 8. Get screenshots/gifs of before and after the fix and attach them to the PR comment.
 
+9. If the testing is unsuccessful and fixes are needed change then remove the label _Needs Testing_ and add _Needs Review_.
```

Comment:
Suggestion: If the testing is unsuccessful and fixes are needed to change, then remove the label `Needs Testing` and add label `Needs Review`